packages/cardano-services/src/TxWorker/utils.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable no-use-before-define */
+import { CONNECTION_ERROR_EVENT, TxSubmitWorker } from '@cardano-sdk/rabbitmq';
+import { CommonOptionDescriptions } from '../ProgramsCommon';
+import { DnsResolver, MissingProgramOption, ServiceNames, srvRecordToRabbitmqURL } from '../Program';
+import { Logger } from 'ts-log';
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import { TxWorkerOptions } from './loadTxWorker';
+
+type WorkerFactory = () => Promise<TxSubmitWorker>;
+
+export const createWorkerFactoryWithDiscovery =
```

Comment:
Please add docs to the exported functions in this file

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/344/commits/89a1a0d3cf617df8746f6493e70edfa219226835